source/using-the-terminal-and-git.html.md.erb

```diff
@@ -1,28 +1,32 @@
-# Cheatsheets
+---
+title: Using the terminal and git
+weight: 70
+---
+
+# Using the terminal and git
 
 ## The terminal
 
 The command line is a text interface for your computer. Just like Windows
-Explorer on Windows or Finder on a Mac it lets you navigate through the files
-and folders of your computer, but it is completely text based.
+Explorer on a PC or Finder on a Mac, it lets you navigate through the files
+and folders of your computer, but it's completely text-based.
 
-The command line can seem unfamiliar and scary, but it's really just a
+The command line can seem unfamiliar and scary, but it's really a
 different way of interacting with your computer. This tutorial only covers
-safe commands that won't do anything bad to your computer, even if you get
+safe commands that will not do anything bad to your computer, even if you get
 them wrong.
 
-*Windows users: Terminal is only available in OS X, for Windows use [Git Bash](https://git-scm.com/download/win
-)*.
+Terminal is only available in Mac OS X. For Windows, use [Git Bash](https://git-scm.com/download/win).
 
 ### Navigating around in the terminal
 
-Once you opened up your terminal you should see a window like this:
+Once you've opened your terminal you should see a window.
 
-![Screenshot of the Terminal app. The text says Last login: Sat May 18 11:08:55 on ttys002 ~$ ](/images/terminal.png)
+![Screenshot of the Terminal window. The text says 'Last login: Sat May 18 11:08:55 on ttys002 ~$'](/images/terminal.png)
 
-Don't worry if the text in yours is a little different, it doesn't matter.
+Do not worry if the text in yours is a little different. It does not matter.
 
-To run a command, just type the words and press `Enter`.
+To run a command, type the words and press `Enter`.
 
 Try running:
```

Comment:
Is there something missing after 'Try running:'?

Reply:
Ah no sorry - there is code underneath. I don't think it showed in the git diff view because it didn't change in this commit.

Reply:
Ah, ok. That makes sense.